src/main/java/uk/gov/justice/digital/hmpps/whereabouts/model/LocationGroup.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.justice.digital.hmpps.whereabouts.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Cell Locations are grouped for unlock lists as a 2 level tree. The two levels are referred to as Location and Sub-Location in the prisonstaffhub UI. Each (location/sub-location) group has a name that is understood by prison officers and also serves as a key to retrieve the corresponding Cell Locations and information about their occupants.
+ **/
+@SuppressWarnings("unused")
```

Comment:
what's unused?

Reply:
My brain when I copy and pasted this from elite2-api